src/main/java/bio/terra/app/usermetrics/UserLoggingMetrics.java

```diff
@@ -11,10 +15,10 @@
  * API request will get grouped together even if they are set in different methods.
  */
 @Component
+@Scope(value = SCOPE_REQUEST, proxyMode = ScopedProxyMode.TARGET_CLASS)
```

Comment:
`@RequestScope` is an alias for the scope and target_class
```suggestion
@RequestScope
```